File: python/AnggotaDPR.py
class AnggotaDPR:
    def __init__(self, id=0, nama="", bidang="", partai=""):
        self.id = id
        self.nama = nama
        self.bidang = bidang
        self.partai = partai

    def setId(self, id):
        self.id = id

    def setNama(self, nama):
        self.nama = nama

    def setBidang(self, bidang):
        self.bidang = bidang

    def setPartai(self, partai):
        self.partai = partai

    def ambilId(self):
        return self.id

    def ambilNama(self):
        return self.nama

    def ambilBidang(self):
        return self.bidang

    def ambilPartai(self):
        return self.partai

File: php/AnggotaDPR.php
<?php

class AnggotaDPR {
    private int $id;
    private string $nama;
    private string $bidang;
    private string $partai;
    private string $photoUrl;

    public function __construct(int $id = 0, string $nama = "", string $bidang = "", string $partai = "", string $photoUrl = "") {
        $this->id = $id;
        $this->nama = $nama;
        $this->bidang = $bidang;
        $this->partai = $partai;
        $this->photoUrl = $photoUrl;
    }

    public function setId(int $id) {
        $this->id = $id;
    }

    public function setNama(string $nama) {
        $this->nama = $nama;
    }

    public function setBidang(string $bidang) {
        $this->bidang = $bidang;
    }

    public function setPartai(string $partai) {
        $this->partai = $partai;
    }

    public function setPhotoUrl(string $photoUrl) {
        $this->$photoUrl = $photoUrl;
    }

    public function ambilId() {
        return $this->id;
    }

    public function ambilNama() {
        return $this->nama;
    }

    public function ambilBidang() {
        return $this->bidang;
    }

    public function ambilPartai() {
        return $this->partai;
    }

    public function ambilPhotoUrl() {
        return $this->photoUrl;
    }
}

?>
